lib/samples/display_clusters/display_clusters.dart

```diff
@@ -143,45 +149,79 @@ class _DisplayClustersState extends State<DisplayClusters>
     // Get the list of geoelements associated with the aggregate geoelement.
     final geoElements = await aggregateGeoElement.getGeoElements();
     // Display a dialog with information about the geoelements.
-    showResultsDialog(geoElements);
+    showOutputs(popup, geoElements);
   }
 
-  void showResultsDialog(List<GeoElement> geoElements) {
-    // Create a dialog that lists the count and names of the provided list of geoelements.
-    showDialog<void>(
+  // Display the results in a Popup view in a modal bottom sheet.
+  void showOutputs(Popup popup, List<GeoElement> geoElements) {
+    final theme = Theme.of(context);
+    final bottomSheetTheme = theme.copyWith(
+      tabBarTheme: theme.tabBarTheme.copyWith(
+        labelColor: theme.colorScheme.primary,
+        unselectedLabelColor: theme.colorScheme.onSurfaceVariant,
+        indicatorColor: theme.colorScheme.primary,
+      ),
+    );
```

Comment:
Looks way cleaner! Hope you agree. Only question is whether all this styling is required? From what I can see the PopupView is displaying it's default style, as is the geoelements tab. So not sure these are doing anything? But I may be wrong!